packages/std/src/init_handle.rs

```diff
@@ -165,6 +187,21 @@ where
     }
 }
 
+impl<T> TryFrom<Context<T>> for HandleResponse<T>
+where
+    T: Clone + fmt::Debug + PartialEq + JsonSchema,
+{
+    type Error = StdError;
+
+    fn try_from(ctx: Context<T>) -> Result<Self, Self::Error> {
+        Ok(HandleResponse {
+            messages: ctx.messages,
+            log: ctx.log,
+            data: ctx.data,
+        })
+    }
+}
```

Comment:
Since this cannot fail we can implement `From<Context<T>> for HandleResponse<T>`. Or do we want the same conversion call in all cases?

Reply:
We could. I though it was cleaner to consistently do `ctx.try_into()` rather than sometime `Ok(ctx.into())`

But just personal preference, either way is fine

Reply:
I implement both now.

Reply:
Ahh... apparently the std lib implements `try_from` if you implement `from`:

```
    = note: conflicting implementation in crate `core`:
            - impl<T, U> std::convert::TryFrom<U> for T
              where U: std::convert::Into<T>;
```